src/README.md

```diff
@@ -0,0 +1,51 @@
+# Editorial principles
+
+## Who is this book for?
+
+## Contents of sections
+
+### Introduction
+
+> If I know nothing about the topic and read this paragraph I will know a
+> surface description of the problem type and the name of the solution approach.
+
+### Typical Problem
+
+> A brief (but comprehensive) description of a typical problem. If I do not know
+> anything about the subject matter I can understand the problem and the problem
+> parameters. If I **do** know the subject matter I should be able to solve the
+> problem.
+
+### Theory
+
+- Discuss some theoretic principles (some basic technical language is acceptable)
+- Use a smaller example (different to the typical problem) that can be done by
+  hand.
+- Some theoretic concepts might be illustrated using the typical problem.
```

Comment:
```suggestion
- Typically there are two things to cover in this section (these might not necessarily be separate): 
    1. A  comprehensive (but not in depth) overview of the theory.
    2. How the theory applies to our typical problem
- Can assume a secondary education in mathematics.
- Concepts that are taught in a typical University mathematics course can be assumed but should include reference texts as **footnotes**.
-  Concepts that are **potentially** not taught in either of the above should be introduced with gentle language, including potential subtle points at programming section (eg "this can be done numerically") and reference texts included as **footnotes**.
- Use a smaller example (different to the typical problem) that can be done by
  hand.
- Some theoretic concepts might be illustrated using the typical problem.
```